lib/DxilDia/DxcPixTypes.cpp

```diff
@@ -14,6 +14,13 @@
 #include "DxcPixTypes.h"
 #include "DxilDiaSession.h"
 
+static const char * GetTypeNameOrDefault(llvm::DIType *diType) {
```

Comment:
Given that every use of this converts the output from ascii to widechar, perhaps this should return widechar instead?

Reply:
Good call. Thanks.